### DISCUSSION TITLE
How can we make nunit-console longPathAware

### DISCUSSION THREAD

**Discussion by sandeep-dhaliwal** [nunit/nunit-console#1365] [Q&A]
manifest file was added to nunit-console here.
https://github.com/nunit/nunit/pull/296/commits/6741163e41ce6cbefc748c62f39e1068266ab6bc

How can we provide our own manifest file? or provide it setting to run with longPathAware option so that tests don't fail when code try to create file with very long name? We could fix it in code everywhere to deal with it or provide this option in manifest file is easier fix if possible.

**Comment by OsirisTerje** [comment]
Why would you do that?

Check this issue for ideas around long paths:  https://github.com/nunit/nunit/issues/4353

**Reply by sandeep-dhaliwal** [reply]
Hi Osiris,

The application takes project as input and write some log/db files while running. If the provided path was too long, it is going to throw these exceptions. We could fix it in code everywhere to deal with it or nunit can provide this option in manifest file which is easier fix and fix every case.
Thanks.

**Reply by OsirisTerje** [reply]
Ok, nice.  An issue around the same is here:  https://github.com/nunit/nunit-console/issues/1367

Further, we have currently 3 versions of the console in different states,  3.15.4,  3.16.3 and 4.0-alpha (main branch).  A fix has to be in all of these, and we don't plan on any new releases in the near future.  

Which version are you using?

And, anything preventing you from using `dotnet test` ?
